### PR TITLE
add veterans topic to index

### PR DESCRIPTION
**Pull request by chejennifer**
diff_report: https://gist.githubusercontent.com/chejennifer/46aba5cb393e634774afff3e8829349c/raw/c6c89983c3a346df32ea587953c52294f01c3068/diff_report.html

Note: not sure if we want ALL the veterans topics to be showing up under veteran unemployment ..?
Screenshot 2023-10-04 at 2 24 26 PM

### CODE REVIEW

File: tools/nl/embeddings/data/curated_input/sheets_svs.csv [hunk 1]
```diff
@@ -1571,3 +1571,8 @@ Exports_EconomicActivity_ElectronicComponents,Electronics Components Exports,,,e
 Imports_EconomicActivity_ElectronicComponents,Electronics Components Imports,,,import of electronics;electronic components import; imports of electronic components;
 Exports_EconomicActivity_Pharmaceutical,Pharmaceutical Exports,,,export of pharmaceuicals;pharmaceutical export; exports of pharmaceuticals;
 Imports_EconomicActivity_Pharmaceutical,Pharmaceutical Imports,,,import of pharmaceuicals;pharmaceutical import; imports of pharmaceuticals;
+dc/topic/Veterans,Veterans,Veterans,,
```
Comment (pradh) [inline tools/nl/embeddings/data/curated_input/sheets_svs.csv:1574]: Could you add some alternate descriptions like how one would query for this in NL? To the Curated_Alternatives column

**Review by pradh — COMMENTED** [review]
One local explore, could you try a few queries and see how it works?  Like

"what are the top wars that the US veterans have fought in?"
"how much do veterans earn in the US?"
"Veterans in which US counties earn the most?"
"what is the gender breakdown of US veterans?"

File: tools/nl/embeddings/data/curated_input/sheets_svs.csv (hunk 1 above)
Comment (chejennifer) [reply tools/nl/embeddings/data/curated_input/sheets_svs.csv]: done

**Comment by chejennifer** [comment]
> On local explore, could you try a few queries and see how it works? Like
> 
> "what are the top wars that the US veterans have fought in?" "how much do veterans earn in the US?" "Veterans in which US counties earn the most?" "what is the gender breakdown of US veterans?"

Tried some queries and somethings I noticed:
- for the first 3 queries, the result always had the correct topic as the first result, but still showed the rest of the topics on the page .. not sure if that's something that would be expected? 
e.g., 
[screen-recording - 2023-10-04T225603.438.webm](https://github.com/datacommonsorg/website/assets/69875368/2c196f5a-eac7-4c3e-b091-6e94b1a49edd)
- for the gender breakdown query, it's not returning the veterans by gender as the top result and I'm slightly puzzled ..
    -  the debug result: https://screenshot.googleplex.com/Byvmi3PVX2x7dMY
    - the result page: 
[screen-recording - 2023-10-04T230042.335.webm](https://github.com/datacommonsorg/website/assets/69875368/ab8046f4-e8f7-4be3-bf2e-fc53b819e012)

**Comment by pradh** [comment]
> > On local explore, could you try a few queries and see how it works? Like
> > "what are the top wars that the US veterans have fought in?" "how much do veterans earn in the US?" "Veterans in which US counties earn the most?" "what is the gender breakdown of US veterans?"
> 
> Tried the example queries and a few things:
> 
> * for the first 3 queries, the result always had the correct topic as the first result, but still showed the rest of the topics on the page .. is that expected?
>   e.g.,
>   [screen-recording - 2023-10-04T225603.438.webm](https://github.com/datacommonsorg/website/assets/69875368/2c196f5a-eac7-4c3e-b091-6e94b1a49edd)

I think that's expected.  We'll show charts for multiple topics.

> * for the gender breakdown query, it's not returning the veterans by gender as the top result and I'm slightly puzzled ..
>   
>   * the debug result: https://screenshot.googleplex.com/Byvmi3PVX2x7dMY
>   * the result page:
>     [screen-recording - 2023-10-04T230042.335.webm](https://github.com/datacommonsorg/website/assets/69875368/ab8046f4-e8f7-4be3-bf2e-fc53b819e012)

That is indeed puzzling!  Maybe we can debug better after you submit this?

**Comment by chejennifer** [comment]
ok, I will submit, thanks for the reviews! & correct, no results when running ./run_test.sh -g

**Pull request merged**